packages/angular-material/src/other/table.renderer.ts

```diff
@@ -87,6 +113,10 @@ export class TableRenderer extends JsonFormsArrayControl {
   mapAdditionalProps(props: ArrayControlProps) {
     this.items = this.generateCells(props.schema, props.path);
     this.displayedColumns = this.items.map(item => item.property);
+    console.log(this.displayedColumns, this.items );
```

Comment:
Please remove this log again. I guess it was leftover from testing ;)

Reply:
Removed